bin/rust-petstore.sh

```diff
@@ -26,7 +26,12 @@ then
 fi
 
 # if you've executed sbt assembly previously it will use that instead.
-export JAVA_OPTS="${JAVA_OPTS} -XX:MaxPermSize=256M -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="generate -t modules/openapi-generator/src/main/resources/rust -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g rust -o samples/client/petstore/rust -DpackageName=petstore_client $@"
+export JAVA_OPTS="${JAVA_OPTS} -Xmx1024M -DloggerPath=conf/log4j.properties"
 
+# Generate client using hyper lib.
+ags="generate -t modules/openapi-generator/src/main/resources/rust -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g rust -o samples/client/petstore/rust -DpackageName=petstore_client --library=hyper $@"
 java ${JAVA_OPTS} -jar ${executable} ${ags}
+
+# Generate client using reqwest lib.
+ags="generate -t modules/openapi-generator/src/main/resources/rust -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g rust -o samples/client/petstore/rust-reqwest -DpackageName=petstore_client --library=reqwest $@"
```

Comment:
Usually, we create another script (e.g. `bin/rust-petstore-reqwest.sh`) for a different HTTP library. (You will find a lot of `./bin/java-petstore-{library}.sh` scripts for different HTTP libraries supported by the Java client generator)

I suggest we do the same for Rust reqwest instead of putting both into one singel script.